src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/IncentiveLevelService.kt

```diff
@@ -0,0 +1,34 @@
+package uk.gov.justice.digital.hmpps.incentivesapi.service
+
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.flow.toList
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.incentivesapi.jpa.IncentiveLevel
+import uk.gov.justice.digital.hmpps.incentivesapi.jpa.repository.IncentiveLevelRepository
+import uk.gov.justice.digital.hmpps.incentivesapi.dto.IncentiveLevel as IncentiveLevelDTO
+
+@Service
+class IncentiveLevelService(
+  private val incentiveLevelRepository: IncentiveLevelRepository,
+) {
+  suspend fun getIncentiveLevels(): List<IncentiveLevelDTO> {
```

Comment:
I would personally add the word `All` to this method.
To make it more explicit it's returning all levels including non-active (without having to look at its definition), e.g. `getAllIncentiveLevels()`.

Reply:
✅